package.json

```diff
@@ -59,8 +59,11 @@
         "copy:flow": "flow-copy-source --ignore \"**/__tests__/**\" --ignore \"**/__mocks__/**\" --ignore \"**/*.ts\" ./src ./es && copyfiles './src/**/*.js.flow' es --up 1",
         "copy:styles": "copyfiles './src/**/*.scss' es --up 1",
         "cy:open": "yarn cy:wait; yarn cypress open",
+        "cy:open:sbk": "yarn cy:wait:sbk; yarn cypress open",
         "cy:run": "yarn cy:wait; yarn cypress run --spec \"test/integration/**/*.cy.js\"",
+        "cy:run:sbk": "yarn cy:wait:sbk; yarn cypress run --spec \"test/integration/**/*.e2e.cy.js\"",
         "cy:wait": "wait-on http-get://localhost:6060/#",
+        "cy:wait:sbk": "wait-on http-get://localhost:6061",
```

Comment:
i think its safe to remove the old cypress commands

Reply:
i think it is too, but, from a purely pragmatic standpoint it seems safer to do additive changes in one step, and then all the removals in a separate step.